### PR TITLE
[WIP] Verify streamed orderbook

### PR DESCRIPTION
**Pull request by nlordell**
This PR adds a new script a streamed orderbook and periodically checks that it matches the on-chain version of the orderbook.

This will hopefully give us some confidence about the new streamed orderbook implementation.

### Test Plan

It is the test!
```
npx truffle exec --network mainnet scripts/verify_streamed_orderbook.js
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+RUN apk add --no-cache --virtual build-dependencies bash git python make g++ ca-certificates
+
+COPY yarn.lock package.json ./
+RUN yarn install --frozen-lockfile && yarn cache clean
+
+COPY . .
+RUN yarn prepack
+
+ENTRYPOINT ["bash", "-c"]
+CMD ["npx", "truffle", "exec", "scripts/verify_streamed_orderbook.js"]
```
Comment (bh2smith) [inline Dockerfile]: No newline at end of file!?

**Review by bh2smith — APPROVED** [review]
Really cool. The terminology used reminds me of a pure mathematical construct I studied extensively in grad school. That is, ["Fibre Bundles"](https://en.wikipedia.org/wiki/Fiber_bundle) or more exotically, [Vector](https://en.wikipedia.org/wiki/Vector_bundle) and [Principal](https://en.wikipedia.org/wiki/Principal_bundle) bundles

File: Dockerfile (hunk 1 above)
Comment (nlordell) [reply Dockerfile]: There is one... When there isn't there is a little symbol: 
image

**Comment by nlordell** [comment]
> Fiber bundles

A "fiber" is [lightweight thread of execution](https://en.wikipedia.org/wiki/Fiber_(computer_science)) made up of promises. This is different than a background thread which would require a `Worker` and is more heavy-weight.

File: Dockerfile (hunk 1 above)
Comment (bh2smith) [reply Dockerfile]: Nice symbol!

**Pull request closed without merging**